src/hotspot/share/jvmci/jvmciCompilerToVM.cpp

```diff
@@ -592,7 +594,7 @@ C2V_VMENTRY_NULL(jobject, lookupType, (JNIEnv* env, jobject, jstring jname, ARGU
   if (val != nullptr) {
     if (strstr(val, "<trace>") != nullptr) {
       tty->print_cr("CompilerToVM.lookupType: %s", str);
-    } else if (strstr(val, str) != nullptr) {
+    } else if (strstr(str, val) != nullptr) {
```

Comment:
This fixing an existing bug: we want to see if `val` is a substring of `str`, not the other way around.